src/UriString.php

```diff
@@ -406,7 +406,7 @@ private static function filterRegisteredName(string $host): string
         $idn_support = $idn_support ?? function_exists('idn_to_ascii') && defined('INTL_IDNA_VARIANT_UTS46');
         // @codeCoverageIgnoreEnd
 
-        $formatted_host = rawurldecode(strtolower($host));
```

Comment:
why was this change added ? What is the relation with the php-cs-fixer issue ?

Reply:
To run tests on Windows You need at first place fix command in composer.json. If there is semicolon at end of command this error is show:
```
[PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]
  The rules contain risky fixers (psr0, psr4), but they are not allowed to run. Perhaps you forget to use --allow-risky=yes option?
```
Code do not recognize options correctly.
When tests runs, they output errors describer in PR on Windows. Those errors are because strtolower function behave different on Windows and Linux. The example code:
```php
$test = '％００.com';
echo $test;
echo strtolower($test);
echo rawurldecode(strtolower($test));
```
produce on Linux:
```
％００.com
％００.com
％００.com
```
and on Windows:
```
％００.com
ﾅﾐﾐ.com
ﾅﾐﾐ.com
```

While on Linux it's still invalid domain name, on Windows it's can be converted to punny-code.